DigitalLearningSolutions.Web/Views/Shared/SearchablePage/_BottomPagination.cshtml

```diff
@@ -3,5 +3,5 @@
 
 <form method="get">
   <partial name="SearchablePage/_SearchSortFilterInputsForPagination" model="Model" />
-  <partial name="PaginatedPage/_TopPaginationNavWithoutAlert" model="Model" />
+  <partial name="PaginatedPage/_BottomPaginationNav" model="Model" />

```

Comment:
I'm going to guess that the weird change of this file going from "TopPaginationWithoutAlert" to "BottomPagination" is just because it is a very similar file to "BottomPaginationWithAlert" which changed to "TopPagination"? Basically want to double check that the right files have top and bottom paginations in the right place.

Reply:
Yeah it's just github being dumb
![image](https://user-images.githubusercontent.com/59561751/167380695-4416dac4-0e30-4944-9c63-d98f84159278.png)